builtin/game/chat.lua

```diff
@@ -774,157 +773,159 @@ core.register_on_punchnode(function(pos, node, puncher)
+		return true, S("Punch a node (range=@1, seconds=@2s, limit=@3).",
+				range, seconds, limit)
 	end,
 })
 
 core.register_chatcommand("rollback", {
-	params = "(<name> [<seconds>]) | (:<actor> [<seconds>])",
-	description = "Revert actions of a player. Default for <seconds> is 60",
+	params = S("(<name> [<seconds>]) | (:<actor> [<seconds>])"),
+	description = S("Revert actions of a player. Default for <seconds> is 60"),
 	privs = {rollback=true},
 	func = function(name, param)
 		if not core.settings:get_bool("enable_rollback_recording") then
-			return false, "Rollback functions are disabled."
+			return false, S("Rollback functions are disabled.")
 		end
 		local target_name, seconds = string.match(param, ":([^ ]+) *(%d*)")
 		if not target_name then
 			local player_name
 			player_name, seconds = string.match(param, "([^ ]+) *(%d*)")
 			if not player_name then
-				return false, "Invalid parameters. See /help rollback"
-						.. " and /help rollback_check."
+				return false,
+				S("Invalid parameters. See /help rollback and /help rollback_check.")
 			end
 			target_name = "player:"..player_name
 		end
 		seconds = tonumber(seconds) or 60
-		core.chat_send_player(name, "Reverting actions of "
-				.. target_name .. " since "
-				.. seconds .. " seconds.")
+		core.chat_send_player(name,
+		S("Reverting actions of @1 since @2 seconds.",
+			target_name, seconds))
 		local success, log = core.rollback_revert_actions_by(
 				target_name, seconds)
 		local response = ""
 		if #log > 100 then
-			response = "(log is too long to show)\n"
+			response = S("(log is too long to show)").."\n"
 		else
 			for _, line in pairs(log) do
 				response = response .. line .. "\n"
 			end
 		end
-		response = response .. "Reverting actions "
-				.. (success and "succeeded." or "FAILED.")
+		if success then
+			response = response .. S("Reverting actions succeeded.")
+		else
+			response = response .. S("Reverting actions FAILED.")
+		end
 		return success, response
 	end,
 })
 
 core.register_chatcommand("status", {
-	description = "Show server status",
+	description = S("Show server status"),
 	func = function(name, param)
 		local status = core.get_server_status(name, false)
 		if status and status ~= "" then
 			return true, status
 		end
-		return false, "This command was disabled by a mod or game"
+		return false, S("This command was disabled by a mod or game.")
 	end,
 })
 
 core.register_chatcommand("time", {
-	params = "[<0..23>:<0..59> | <0..24000>]",
-	description = "Show or set time of day",
+	params = S("[<0..23>:<0..59> | <0..24000>]"),
+	description = S("Show or set time of day"),
 	privs = {},
 	func = function(name, param)
 		if param == "" then
 			local current_time = math.floor(core.get_timeofday() * 1440)
 			local minutes = current_time % 60
 			local hour = (current_time - minutes) / 60
-			return true, ("Current time is %d:%02d"):format(hour, minutes)
+			return true, S("Current time is @1:@2",
+				tostring(hour),
```

Comment:
(unrelated, but since you are touching it anyway)
```suggestion
				string.format("%02d", hour),
```